build/Targets/Packages.props

```diff
@@ -3,220 +3,222 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Why did this file change so much?  No way to review it.

Did you edit this on unix and as such change line endings?  If so can you change ti back so we can see the diff?